graphistry/compute/hop.py

```diff
@@ -205,9 +250,25 @@ def hop(self: Plottable,
         hop_edges_forward = None
         new_node_ids_forward = None
         if direction in ['forward', 'undirected']:
+            # Prepare edges for forward merging - handle column name conflicts
+            if node_src_conflict:
+                # When node and source columns have the same name, use a temporary column
+                edges_for_merge = edges_indexed.copy()
```

Comment:
This may reduce performance, is there a way to solve conflicts without creating/copying to a temp object?